[datadog] Update agent to v5.11.2

johnaxel opened the pull request:
Updates the version of the Datadog Agent on Universe and enables service discovery by default for simpler monitoring of Docker containers.

hkaj commented:
LGTM 👍

ryadav88 commented on repo/packages/D/datadog/2/package.json:
```diff
@@ -0,0 +1,23 @@
+{
+    "packagingVersion": "3.0",
+    "description": "Cloud-scale monitoring",
```
Can you add more details to the description

ryadav88 commented on repo/packages/D/datadog/2/package.json:
```diff
@@ -0,0 +1,23 @@
+    "name": "datadog",
+    "postInstallNotes": "Datadog on DCOS successfully installed.\nHead over to https://app.datadoghq.com to monitor your DCOS cluster.",
+    "postUninstallNotes": "Datadog on DCOS successfully uninstalled.\n.",
+    "preInstallNotes": "Pass your API key as a package option (datadog.api_key).\nAlso pass the number of Mesos slaves as an option (datadog.instances).\nMarathon will make sure that the Datadog Agent runs only once per slave.",
```
Can you add a blurb about - where should users go to get the API key

johnaxel commented:
Hi @ryadav88, I think I've addressed your feedback here, and have opened a PR with a getting-started doc for the package:

https://github.com/dcos/examples/pull/104
